meta.yaml

```diff
@@ -20,7 +20,7 @@ requirements:
     - python {{ python }}
     - joblib
     - numpy>=1.16.0
-    - pandas>=0.24
+    - pandas<=0.25
```

Comment:
Same for environment.yml and tox? Actually, it would be great if you can fix the issues with pandas v1 very quickly.

Reply:
Done, thanks. I'll try to do it soon (or even better, find someone who wants to do it).

Reply:
Nit-picky here, but don't you want
```
- pandas>=0.24,<1.0
```
here to be on the safe side? https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#package-match-specifications

Reply:
Yes. Just didn't know that this works ;-)